### DISCUSSION TITLE
Will version 0.10.0 be released soon?

### DISCUSSION THREAD

**Discussion by joaodinissf** [eclipse/tm4e#697] [General]
Hi there,

Thank you for the many improvements that have been committed in the last few days. A project I am collaborating on would benefit greatly from incorporating these improvements.

Are you planning to release version 0.10.0 soon?

Thanks in advance.

**Comment by sebthom** [comment]
I am currently trying to add support for .tmThemes. Once that is done I create a release. In any case I want a release before 2024-03 so it can be included in the Simultaneous Release train

**Comment by sebthom** [comment]
0.10.0 is now released.
